### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by RareHelix**
- [DEMO LINK](https://RareHelix.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://RareHelix.github.io/layout_moyo-header/report/html_report/)
- 
 Header height is set in 1 place (for the links)
 Content is vertically centered (for any header height). - [x]
 CSS is used to show all letters in Uppercase (don't type them in HTML) - [x]
 Logo is an image wrapped with a link - [x]
 CSS Variable is used for a blue color
 Pseudo-element is used for a blue line below the active link - [x]
 Code follows all the [Code Style Rules ❗️] - [x](https://github.com/RareHelix/layout_moyo-header/blob/master/checklist.md)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,85 @@
...
+  color: #00acdc;
+}
+
+a[data-qa]:hover {
+  color: #00acdc;
+}
+
+/* [data-qa='hover'] {
+  color: #00acdc;
+} */
```
Comment (anastasiiavorobiova) [inline src/style.css:85]:
```suggestion

```

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,85 @@
...
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.is-active::after {
+  color: #00acdc;
```
Comment (anastasiiavorobiova) [inline src/style.css:60]: Consider using variables for repeating colors

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

**Review by RareHelix — COMMENTED** [review]
Змінила ноуд на 20, можливо через це не пропускає

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,85 @@
...
+  left: 0;
+  right: 0;
+  height: 4px;
+  border-radius: 2px;
+  background-color: var(--main-color);
+}
+
+a.is-active {
+  color: var(--main-color);
+}
```
[the remark this answers is not in the document]
Comment (RareHelix) [reply src/style.css:77]: Не розумію,що тут виправити?

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,85 @@
...
+
+.nav_link:hover {
+  color: var(--main-color);
+}
+
+a[data-qa]:hover {
+  color: var(--main-color);
+}
```
[the remark this answers is not in the document]
Comment (RareHelix) [reply src/style.css:85]: Не розумію,що тут виправити?

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,85 @@
+:root {
+  --main-color: #00acdc;
+}
+
+html {
+  margin: 0;
+}
```
[the remark this answers is not in the document]
Comment (RareHelix) [reply src/style.css:7]: Не розумію,що тут виправити?

[the remark this answers is not in the document]
Comment (etojeDenys) [reply src/style.css:7]: видалити цей селектор

File: src/style.css (hunk 3 above)
[the remark this answers is not in the document]
Comment (etojeDenys) [reply src/style.css:77]: видалити теж

File: src/style.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (etojeDenys) [reply src/style.css:85]: видалити

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Review by etojeDenys — CHANGES_REQUESTED** [review]
you still haven't fix all prev comments
image

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, you need to fix all comments from the previous review, if you need help feel free in the fe_chat

**Comment by RareHelix** [comment]
I make changes and write, but they are not saved for some reason

вт, 10 вер. 2024 р. о 11:02 Volodymyr Soltys ***@***.***>
пише:

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,74 @@
...
+.nav_link {
+  display: block;
+  color: black;
+  text-decoration: none;
+  position: relative;
+  line-height: 60px;
+  text-transform: uppercase;
+}
+
+.nav_list:first-child {
```
Comment (anastasiiavorobiova) [inline src/style.css:48]:
```suggestion
.nav__item:first-child {
```

**Review by anastasiiavorobiova — APPROVED** [review]
Great work!

**Pull request closed without merging**